### ISSUE TITLE
Cannot upload artifacts

### ISSUE THREAD

**Issue by kinosang** [eine/tip#160]
Got error of `ssl.SSLEOFError: EOF occurred in violation of protocol (_ssl.c:2384)`

full logs see https://github.com/JoyMoe/OpenData/runs/3917346331?check_suite_focus=true

**Comment by kinosang** [comment]
@eine after remove the release, it works well now. I'll keep watching for some days.

**Comment by lazka** [comment]
Tried again, still fails :( https://github.com/msys2/msys2-installer/runs/4202718077?check_suite_focus=true

Likely the problem is with github, still annoying :(

**Comment by lazka** [comment]
Could this be an alpine related issue, searching for the SSL error gives me various alpine using users. Maybe we could try the default docker image instead?

**Comment by eine** [comment]
@lazka, absolutely! If that is the case, that'd be an awesome catch! Let me run some local tests and I'll update the dockerfile.

**Comment by eine** [comment]
So, on the one hand I changed the base image from `python:alpine` to `python:slim-bullseye`. That's twice the size (142MB vs 64 MB), but let's hope it solves the problem. On the other hand, I added a variant of the action to subdir `composite`. As the name suggests, that's a Composite Action instead of a Docker Action. That will install the dependencies (PyGitHub) on the workflow, instead of using an isolated container. However, it will let us know whether the problems are due to using containers.

@lazka, @stnolting, @kinosang, if you want to try the composite variant, just changing the name should be enough: `eine/tip/composite@master`.

**Comment by lazka** [comment]
> That's twice the size (142MB vs 64 MB), but let's hope it solves the problem.

Just wanted to point out that despite it being larger it is still faster to run since there are Python wheels for Debian while it compiles packages on alpine.

**Comment by lazka** [comment]
Still the same issue: https://github.com/msys2/msys2-installer/runs/4229934050?check_suite_focus=true

Sorry for the wrong lead :/

**Comment by lazka** [comment]
So, another lead: this started to happen exactly when Python 3.10 was released. And the docker image is always using the newest one.

**Comment by lazka** [comment]
Now it's just another error: https://github.com/msys2/msys2-installer/runs/4273363943?check_suite_focus=true

But at least it only fails every second time instead of every time :) oh well...
